flink-ml-lib/src/main/java/org/apache/flink/ml/common/MapPartitionFunctionWrapper.java

```diff
@@ -0,0 +1,68 @@
+package org.apache.flink.ml.common;
+
+import org.apache.flink.api.common.functions.MapPartitionFunction;
+import org.apache.flink.api.common.functions.RichMapPartitionFunction;
+import org.apache.flink.api.common.functions.util.FunctionUtils;
+import org.apache.flink.api.common.state.ListState;
+import org.apache.flink.api.common.state.ListStateDescriptor;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.runtime.state.StateInitializationContext;
+import org.apache.flink.streaming.api.operators.AbstractStreamOperator;
+import org.apache.flink.streaming.api.operators.BoundedOneInput;
+import org.apache.flink.streaming.api.operators.OneInputStreamOperator;
+import org.apache.flink.streaming.runtime.streamrecord.StreamRecord;
+import org.apache.flink.util.Collector;
+
+/**
+ * MapPartitionFunction wrapper.
+ *
+ * @param <IN> Input element type.
+ * @param <OUT> Output element type.
+ */
+public class MapPartitionFunctionWrapper<IN, OUT> extends AbstractStreamOperator<OUT>
```

Comment:
Can we remove this class and use the existing one `org.apache.flink.ml.common.datastream.MapPartitionFunctionWrapper`?

Reply:
OK, I will refine it.

Reply:
done